events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
-# Communications Lead Handbook
+# Communications Team Handbook
 
 This document defines communications activities needed to run a
-Contributor Summit.
+Kubernetes Contributor Summit (KCS). The Comms Team consists of 1-2 lead(s), shadows, volunteers, at least 1 emeritus lead, and a Website Manager.
 
-- [Overview](#overview)
-- [Skills and Qualifications](#skills-and-qualifications)
-- [Responsibilities](#responsibilities)
-  - [Communication Channels and Property Access](#communication-channels-and-property-access)
+Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.
```

Comment:
> 2 cycles, 1 year

Could we say 2 cycles per year? Is that accurate with India being in the mix now too or is someone else running with it?

Reply:
Also, cycle, I wasn't clear if that meant EU and US but I'm assuming so.